src/Rule/EnsureExplicitNullableTypes.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of DOCtor-RST.
+ *
+ * (c) Oskar Stark <oskarstark@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Rule;
+
+use App\Attribute\Rule\Description;
+use App\Value\Lines;
+use App\Value\NullViolation;
+use App\Value\RuleGroup;
+use App\Value\Violation;
+use App\Value\ViolationInterface;
+
+#[Description('Ensure explicit nullable types in method arguments.')]
+class EnsureExplicitNullableTypes extends AbstractRule implements LineContentRule
+{
+    public static function getGroups(): array
+    {
+        return [
+            RuleGroup::Symfony(),
+        ];
+    }
+
+    public function check(Lines $lines, int $number, string $filename): ViolationInterface
+    {
+        $lines->seek($number);
+        $line = $lines->current();
+
+        if (!str_contains((string) $line->clean(), ' = null')) {
+            return NullViolation::create();
+        }
```

Comment:
To be sure, it's only for optimization right (avoid match when code style is already good) ?

Reply:
Yep

Reply:
Well "not concerned" more than "good" but yes... Do you find it superfluous?

Reply:
No it's fine